### DISCUSSION TITLE
编译的ssrp和passwall都出不去。不知道是不是哪里设置问题，求助

### DISCUSSION THREAD

**Discussion by bonnyandsky** [immortalwrt/immortalwrt#557] [Q&A] [answered]
今天编译的immortalwrt固件
分支：21.02
SNAPSHOT r19086-29c9109246
光猫桥接,路由器拨号
就设置了pppoe拨号，然后设置了下lan相关信息。其他没动什么，设置ssrp和passwaal后都可以订阅节点，但是设置节点后不管是gfw还是ip模式，都出不去。
因为以前一直用lean的源代码编译的,换成21.02不知道是不是哪里设置问题导致的出不去.
------------------------------------------------------------------------------------------------------------------------------
netwrok设置如下:
config interface 'loopback'
	option device 'lo'
	option proto 'static'
	option ipaddr '127.0.0.1'
	option netmask '255.0.0.0'

config globals 'globals'

config device
	option name 'br-lan'
	option type 'bridge'
	list ports 'eth0'
	list ports 'eth2'
	list ports 'eth3'

config interface 'lan'
	option device 'br-lan'
	option proto 'static'
	option ipaddr '192.168.0.1'
	option netmask '255.255.255.0'
	option ip6assign '60'

config interface 'wan'
	option device 'eth1'
	option proto 'pppoe'
	option username '*******'
	option password '*******'
	option ipv6 'auto'

config interface 'wan6'
	option device 'eth1'
	option proto 'dhcpv6'

config interface 'VPN'
	option device 'ipsec0'
	option proto 'static'
	option ipaddr '192.168.0.1'
	option netmask '255.255.255.0'
-------------------------------------------------------------------
dhcp设置如下:
config dnsmasq
	option domainneeded '1'
	option localise_queries '1'
	option rebind_protection '1'
	option rebind_localhost '1'
	option local '/lan/'
	option domain 'lan'
	option expandhosts '1'
	option authoritative '1'
	option readethers '1'
	option leasefile '/tmp/dhcp.leases'
	option resolvfile '/tmp/resolv.conf.d/resolv.conf.auto'
	option localservice '1'
	option dns_redirect '1'
	option ednspacket_max '1232'
	option localuse '1'

config dhcp 'lan'
	option interface 'lan'
	option leasetime '12h'
	option dhcpv4 'server'
	list ra_flags 'managed-config'
	list ra_flags 'other-config'
	option ndp 'hybrid'
	option ra_management '1'
	option start '150'
	option limit '49'
	option force '1'
	option ra 'server'
	option dhcpv6 'server'

config dhcp 'wan'
	option interface 'wan'
	option ignore '1'
	list ra_flags 'none'

config odhcpd 'odhcpd'
	option maindhcp '0'
	option leasefile '/tmp/hosts/odhcpd'
	option leasetrigger '/usr/sbin/odhcpd-update'
	option loglevel '4'
--------------------------------------------------------------------------------------------------------

passwall和ssrp的除了订阅节点后选择了节点，其他设置都是默认没动。模式方面试过全局，gfw，中国列表以外。2个软件都出不去。

**Comment by bonnyandsky** [comment]
![Screenshot_2022-01-05-08-26-13-512_org telegram messenger](https://user-images.githubusercontent.com/30990861/148142844-139fcdaa-ebe5-4eb9-b608-c862053e35da.png)
刚看到说ssrp改了验证设定。难道是这个关系？pw也受影响？

**Comment by chobitsos** [comment, marked as answer]
启动日志丢出来看看

**Reply by bonnyandsky** [reply]
感谢回复。我找到问题原因了。是因为我iptv所用vlan配置不正确导致的。这个我关闭吧。我重新发一个询问设置vlan的。21.02和18.06的vlan设置很不一样。研究了几天 也没个所以然。还是要请各位帮忙啊。
